### PR TITLE
[DOCS-6574] New Resource Catalog page

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
This PR creates a new page for Resource Catalog under Infrastructure, maintaining a lot of the existing content from where it currently [lives](https://docs.datadoghq.com/security/misconfigurations/resource_catalog). 
Adds a new section for Inventory tab.
Adds alias for old page

Adds a small section to main CSM landing page since it still overlaps with CSM (customers can still access the Resource Catalog security tab from within CSM. They however won’t be able to see inventory tab under CSM)

Moves the Cloud Resources Schema also nested under Resource Catalog along with single sourced docs updates needed.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/infrastructure/resource_catalog/_index.md [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
 title: Datadog Resource Catalog
 kind: documentation
+is_beta: true
 aliases:
   - /security_platform/cspm/resource_catalog
   - /security/cspm/resource_catalog
+  - /security/misconfigurations/resource_catalog
 further_reading:
-- link: "/security/misconfigurations/custom_rules/schema/"
+- link: "security/misconfigurations/"
```
Comment (kayayarai) [inline content/en/infrastructure/resource_catalog/_index.md:10]:
```suggestion
- link: "/security/misconfigurations/"
```

File: content/en/infrastructure/resource_catalog/_index.md [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
   tag: "Documentation"
-  text: "Cloud Resource Schema"
+  text: "Cloud Security Management Misconfigurations"
+- link: "security/threats/"
```
Comment (kayayarai) [inline content/en/infrastructure/resource_catalog/_index.md:13]:
```suggestion
- link: "/security/threats/"
```

File: content/en/infrastructure/resource_catalog/_index.md [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
 
 Resource Catalog leverages Datadog cloud integrations and the Datadog Agent to gather data from cloud resources such as databases, storage services, and hosts.
 
-{{< img src="security/cspm/resource_catalog/resource_catalog2.png" alt="Resource Catalog map view displaying host and cloud resources grouped by category and region." style="width:100%;" >}}
+{{< img src="/infrastructure/resource_catalog/Resource_Catalog.png" alt="The Resource Catalog page showing the Inventory tab, sorting by service" width="100%">}}
+
+## Setup
 
-## Adding resources to Resource Catalog
+By default, you can view all resources monitored by the Datadog Agent and other Cloud Integrations. To view additional cloud resources in the Resource Catalog, please enable resource collection from the [Resource Catalog][5] setup page. To gain insights into your security risks, enable [Cloud Security Management][1] for each cloud account.
```
Comment (kayayarai) [inline content/en/infrastructure/resource_catalog/_index.md:36]:
```suggestion
By default, you can view all resources monitored by the Datadog Agent and Cloud Integrations. To view additional cloud resources in the Resource Catalog, enable resource collection from the [Resource Catalog][5] setup page. To gain insights into your security risks, enable [Cloud Security Management][1] for each cloud account.
```

File: content/en/infrastructure/resource_catalog/_index.md [hunk 4]
```diff
@@ -38,8 +50,8 @@ The Resource Catalog is useful for:
 
 On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization. The catalog detects a resource either because it has an Agent installed on it, or because a cloud integration is configured on it. Information about the resources in your organization is shown in the ownership and security tabs, with two views: List and Map.
 
-**Ownership Tab**:
-The ownership tab allows you to understand the resource ownership information such as team, service, on-call, and contact. This page allows users to proactively identify lack of ownership information and resolve it prior to an incident occurring. 
+**Inventory Tab**:
+The inventory tab allows you to understand the context for a resource such as team ownership and related service. This page allows you to proactively identify lack of ownership information and resolve it prior to an incident occurring. The Resource Catalog presents configuration information customized to the resource type being viewed. It also allows you to search resources by specific configuration attributes such as `instance_type` for a host, or a `version` for a database.
```
Comment (kayayarai) [inline content/en/infrastructure/resource_catalog/_index.md:54]:
```suggestion
The Inventory tab shows context for a resource, including team ownership and related services. It helps you proactively identify and provide missing ownership information before you need it in an incident. The Resource Catalog shows configuration information customized for each resource type. You can search resources by specific configuration attributes such as `instance_type` for a host, or a `version` for a database.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just a few comments. The preview seems to work well with the single-sourcing changes you made 👍 . 

One note: it's best practice to name files (in this case, screen caps) all lower-case because occasionally they will pass through systems where filenames get downcased, and you still want it to work. I recommend changing your screen cap file names and the in-page links to them to be consistently lower-case.

File: Makefile [hunk 5]
```diff
@@ -42,6 +42,7 @@ help:
 clean-all: clean clean-examples clean-dependent-repos ## Clean everything (environment, sourced repos, generated files)
 	rm -rf ./node_modules ./hugpython ./public
 
+
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply Makefile:45]: Hmmm... That must be auto-generated or not sure what happened as I didn't do anything with the `Makefile`, I'll take a look.

**Comment by aliciascott** [comment]
> Just a few comments. The preview seems to work well with the single-sourcing changes you made 👍 .
> 
> One note: it's best practice to name files (in this case, screen caps) all lower-case because occasionally they will pass through systems where filenames get downcased, and you still want it to work. I recommend changing your screen cap file names and the in-page links to them to be consistently lower-case.

Ok - got those updated, I did re-name the images in the branch as well to lower-case but it didn't trigger them to be re-added with the new lowercase name, the preview is still fine, but let me know if that is ok.

**Review by kayayarai — APPROVED** [review]
LGTM!

**Pull request merged**